src/main/java/org/jitsi/protocol/xmpp/util/SSRCSignaling.java

```diff
@@ -245,35 +247,42 @@ public static boolean mergeVideoIntoAudio(MediaSSRCMap peerSSRCs)
         if (audioSSRC == null)
             return false;
 
-        // Find first video SSRC with non-empty stream ID and different
-        // than 'default' which is sometimes used when unspecified
-        List<SourcePacketExtension> videoSSRCs
-            = peerSSRCs.getSSRCsForMedia(MediaType.VIDEO.toString());
-
-        SourcePacketExtension videoSSRC = getFirstWithMSID(videoSSRCs);
-        // No video to sync
-        if (videoSSRC == null)
-            return false;
-
-        // Will merge stream by modifying msid and copying cname and label
         String audioStreamId = getStreamId(audioSSRC);
-        String videoTrackId = getTrackId(videoSSRC);
-        ParameterPacketExtension videoMsid = getParam(videoSSRC, "msid");
-
-        if ( StringUtils.isNullOrEmpty(audioStreamId)
-             || StringUtils.isNullOrEmpty(videoTrackId)
-             ||  videoMsid == null )
+        if (audioStreamId == null)
         {
+            // No valid audio stream
             return false;
         }
 
-        // Copy cname and label
-        copyParamAttr(videoSSRC, audioSSRC, "cname");
-        copyParamAttr(videoSSRC, audioSSRC, "mslabel");
+        // Find first video SSRC with non-empty stream ID and different
+        // than 'default' which is sometimes used when unspecified
+        List<SourcePacketExtension> videoSSRCs
+            = peerSSRCs.getSSRCsForMedia(MediaType.VIDEO.toString());
 
-        videoMsid.setValue(audioStreamId + " " + videoTrackId);
+        boolean merged = false;
+        // There are multiple video SSRCs in simulcast
+        // FIXME this will not work with more than 1 video stream
```

Comment:
do RTX SSRCs belong to the same stream (do they have the same msid) ? if the answer is *yes* then it should be fine as it's the same as in simulcast. If you were asking if that would break *before the PR* then yes that could break. So far we were lucky that the client was taking the msid value only from the first SSRC, but I saw it randomly breaking while working on P2P.